config/locales/active_admin.de.yml

```diff
@@ -431,6 +432,12 @@ de:
           notice: "Rückgängig:"
         action_item:
           link_to: "rückgängig"
+      member_action:
+        set_menu_item_online: "Diesen Menüpunkt online schalten."
```

Comment:
Perhaps talking of visibility is more apropriate then of online/offline? Or was this a specific request?